recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: conda
   # VERSION_OVERRIDE is used by the canary release workflow
-  version: {{ os.getenv("VERSION_OVERRIDE") or GIT_DESCRIBE_TAG }}.{{ GIT_BUILD_STR }}
```

Comment:
Alternatively we can replace `.` with `+` to make the build string the local version label

Reply:
The only problem is that we'll still get build failures because hatch will still complain because it's not PEP-440 compliant ☹️

Reply:
The local version tag is PEP440 compliant? https://peps.python.org/pep-0440/#local-version-identifiers

It hatch even more strict?

Reply:
Didn't notice that the first time I skimmed through the PEP. Thanks. Will update this pull request accordingly.